bundles/org.openhab.binding.matter/README.md

```diff
@@ -0,0 +1,472 @@
+Number:Temperature    TestHVAC_HeatSetpoint    "Heat Setpoint [%d °F]"    (TestHVAC)    ["Setpoint", "Temperature"]      {matter="thermostat.occupiedHeatingSetpoint"}
+Number:Temperature    TestHVAC_CoolSetpoint    "Cool Setpoint [%d °F]"    (TestHVAC)    ["Setpoint", "Temperature"]      {matter="thermostat.occupiedCoolingSetpoint"}
+Number                TestHVAC_Mode            "Mode [%s]"                (TestHVAC)    ["Control" ]                     {matter="thermostat.systemMode" [OFF=0, HEAT=1, COOL=2, AUTO=3]}
+
+Switch                TestDoorLock             "Door Lock"                                                               {matter="DoorLock"}
+Rollershutter         TestShade                "Window Shade"                                                            {matter="WindowCovering"}
+Number:Temperature    TestTemperatureSensor    "Temperature Sensor"                                                      {matter="TemperatureSensor"}
+Number                TestHumiditySensor       "Humidity Sensor"                                                         {matter="HumiditySensor"}
+Switch                TestOccupancySensor      "Occupancy Sensor"                                                        {matter="OccupancySensor"}
+
+### Fan with group item control
+Group                 TestFan                  "Test Fan"                                                                {matter="Fan" [fanControl-fanModeSequence=3]}
+Dimmer                TestFanSpeed             "Speed"                    (TestFan)                                      {matter="fanControl.percentSetting"}
+Switch                TestFanOnOff             "On/Off"                   (TestFan)                                      {matter="fanControl.fanMode"}
+Number                TestFanMode              "Mode"                     (TestFan)                                      {matter="fanControl.fanMode" [OFF=0, LOW=1, MEDIUM=2, HIGH=3, ON=4, AUTO=5, SMART=6]}
+
+### Fan with single item control , so no group item is needed
+Switch                TestFanSingleItem         "On/Off"                                                                 {matter="Fan, fanControl.fanMode"}
+```
+
+### Bridge FAQ
+
+* Alexa: When pairing, after a minute Alexa reports "Something went wrong" 
+  * Alexa can take 3-4 seconds per device to process which can take longer then the Alexa UI is willing to wait.  Eventually the pairing will complete, which for a large number of devices may be a few minutes. 
+* Alexa: Suddenly stops working and says it could not connect to a device or device not responding.
+  * Check the Settings page in the Main UI to confirm the bridge is running
+  * Ensure the openHAB item has the proper matter tag, or that the item is being loaded at all (check item file errors)
+  * Rarely, you may need to reboot the Alexa device.  If you have multiple devices and not sure which is the primary matter connection, you may need to reboot all of them.
```

Comment:
```suggestion
- Alexa: When pairing, after a minute Alexa reports "Something went wrong" 
  - Alexa can take 3-4 seconds per device to process which can take longer then the Alexa UI is willing to wait.  Eventually the pairing will complete, which for a large number of devices may be a few minutes. 
- Alexa: Suddenly stops working and says it could not connect to a device or device not responding.
  - Check the Settings page in the Main UI to confirm the bridge is running
  - Ensure the openHAB item has the proper matter tag, or that the item is being loaded at all (check item file errors)
  - Rarely, you may need to reboot the Alexa device.  If you have multiple devices and not sure which is the primary matter connection, you may need to reboot all of them.
```